governance.md

```diff
@@ -90,20 +91,31 @@ available.
 
 ### Steering Committee Officers
 
-Officers consist of a Chair, a Vice-Chair, an Election Chair, a Treasurer, and a Secretary, elected by the Steering
+Officers consist of:
+
+* Chair
+* Vice Chair
+* Election Chair
+* Treasurer
+* Secretary
+* 2 International Council of RSE Associations Representatives
+
+These positions are elected by the Steering
 Committee. Officer elections will be held at the meeting following the election of new members to
 the Steering Committee.
 
 #### Chair
 
-The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice-
-Chair or another Steering Committee member to do so.
+The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice 
+Chair or another Steering Committee member to do so.  
+The Chair is responsible for convening the Annual General Meeting of the membership.  
+The Chair is responsible for reporting on US-RSE activities over the past year at the Annual General Meeting.
```

Comment:
This seems like a duplicate of the Vice Chair duty to "annually prepare a report of Association activities in the past year".

Reply:
Can you suggest a specific change you want to make so we can just commit it?